StreamChat.xcodeproj/project.pbxproj

```diff
@@ -2797,7 +2799,7 @@
 				79CDE1DD24B321FE0003BD1D /* CurrentUserDTO.swift in Sources */,
 				797A756824814F0D003CF16D /* BundleExtensions.swift in Sources */,
 				79877A192498E4EE00015F8B /* UserPayload.swift in Sources */,
-				792A4F1B247FE84900EAF71D /* ChannelQueryUpdater.swift in Sources */,
+				792A4F1B247FE84900EAF71D /* ChannelListQueryUpdater.swift in Sources */,
```

Comment:
This seems like a leftover from some failed conflict merge. Try to rebase with master again.

Reply:
How it should be?